### ISSUE TITLE
Apply-PnPProvisioningTemplate : File Not Found. When provisioning XsltListViewWebPart to a page on the root site collection of a SP Web Application

### ISSUE THREAD

**Issue by JCol79** [SharePoint/PnP-Sites-Core#1560]
#### Which PnP repository should you use to report the issue? 
PnP JS Core - https://github.com/OfficeDev/PnP-JS-Core

#### Category
[ X ] Bug
[ ] Enhancement

#### Environment
[ ] Office 365 / SharePoint Online
[ X ] SharePoint 2016
[ ] SharePoint 2013

If SharePoint on-premises, what's exact CU version:  July 2017 CU

#### Expected or Desired Behavior
To use apply a PnPProvisioningTemplate intended to add an XsltListViewWebPart to a page at any site collection.

#### Observed Behavior
When applying a PnPProvisioningTemplate intended to add an XsltListViewWebPart to a page at the root site collection of a SPWebApplication, it fails both in C# project or using Apply-PnPProvisioningTemplate.

but it works as expected on any other site collection that is NOT the root site collection of the SPWebApplication

#### Steps to Reproduce
Use a template like the one below to apply to both a root site collection of the SPWebApplication (example: http://yourSPserver) and a site collection in a managed path in the SPWebApplication (example: http://yourSPserver/sites/test).

Simple test is to the apply template utilizing Apply-PnPProvisioningTemplate. Ive tested with old and latest version of the cmdlet.

Sample XML PnPTemplate:
<?xml version="1.0"?>
<pnp:Provisioning xmlns:pnp="http://schemas.dev.office.com/PnP/2017/05/ProvisioningSchema">
  <pnp:Preferences Author="WSS" Generator="Human beings :-)"></pnp:Preferences>
  <pnp:Templates ID="testIssue-PnPTemplates">
    <pnp:ProvisioningTemplate ID="testIssue_PnP_testIssueHomepage" >
      <pnp:WebSettings WelcomePage="SitePages/testIssue.aspx"/>
      <pnp:Lists>
        <pnp:ListInstance Title="Announcements" Url="Lists/Announcements" Description="" TemplateType="100"  EnableVersioning="true" Hidden="false" OnQuickLaunch="true">
          <pnp:ContentTypeBindings>
            <pnp:ContentTypeBinding ContentTypeID="0x01" Default="true"/>
            <pnp:ContentTypeBinding ContentTypeID="0x0120" />
          </pnp:ContentTypeBindings>
          <pnp:Fields>
            <Field Type="Note" DisplayName="Body" Required="TRUE" EnforceUniqueValues="FALSE" Indexed="FALSE" NumLines="6" RichText="TRUE" RichTextMode="FullHtml" IsolateStyles="TRUE" Sortable="FALSE" ID="{e5fa403a-5877-403b-bc1d-7066e42b1055}" SourceID="{{listid:Test}}" StaticName="Body" Name="Body" ColName="ntext2" RowOrdinal="0" Version="2" RestrictedMode="TRUE" AppendOnly="FALSE" />
            <Field Type="Text" DisplayName="Description" Required="FALSE" EnforceUniqueValues="FALSE" Indexed="FALSE" MaxLength="255" ID="{32013cdb-83e0-4868-9115-7b984cc2ca62}" SourceID="{{listid:Test}}" StaticName="Description" Name="Description" ColName="nvarchar3" RowOrdinal="0" Version="1" />
            <Field Type="DateTime" DisplayName="BeginDate" Required="TRUE" EnforceUniqueValues="FALSE" Indexed="FALSE" Format="DateOnly" FriendlyDisplayFormat="Disabled" ID="{22cf99e9-dc87-4334-8409-ac1d454beaf7}" SourceID="{{listid:Test}}" StaticName="BeginDate" Name="BeginDate" ColName="datetime1" RowOrdinal="0" Version="1" />
            <Field Type="DateTime" DisplayName="Expires" Required="TRUE" EnforceUniqueValues="FALSE" Indexed="FALSE" Format="DateOnly" FriendlyDisplayFormat="Disabled" ID="{13e9017c-efcb-4ae2-bef7-d6663e098708}" SourceID="{{listid:Test}}" StaticName="Expires" Name="Expires" ColName="datetime2" RowOrdinal="0" Version="2" CalType="0" />
          </pnp:Fields>
          <pnp:Views>
            <View Name="{48B9C0A7-193D-49F0-AE21-5FB76AE05CD9}" DefaultView="TRUE" MobileView="TRUE" MobileDefaultView="TRUE" Type="HTML" DisplayName="All Items" Url="/Lists/Announcements/AllItems.aspx" Level="1" BaseViewID="1" ContentTypeID="0x" ImageUrl="/_layouts/15/images/generic.png?rev=40">
              <Query>
                <OrderBy>
                  <FieldRef Name="ID" />
                </OrderBy>
              </Query>
              <ViewFields>
                <FieldRef Name="LinkTitle" />
                <FieldRef Name="Body" />
                <FieldRef Name="Description" />
                <FieldRef Name="BeginDate" />
                <FieldRef Name="Expires" />
              </ViewFields>
              <RowLimit Paged="TRUE">30</RowLimit>
              <JSLink>clienttemplates.js</JSLink>
            </View>
          </pnp:Views>
        </pnp:ListInstance>
      </pnp:Lists>
      <pnp:Pages>
        <pnp:Page Layout="TwoColumns" Overwrite="true" Url="{site}/SitePages/testIssue.aspx">
          <pnp:WebParts>
            <pnp:WebPart Title="Announcements" Row="1" Column="1">
              <pnp:Contents>
                <webParts>
                    <webPart xmlns="http://schemas.microsoft.com/WebPart/v3">
                      <metaData>
                        <type name="Microsoft.SharePoint.WebPartPages.XsltListViewWebPart,Microsoft.SharePoint, Version=15.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c" />
                        <importErrorMessage>Cannot import this Web Part.</importErrorMessage>
                      </metaData>
                      <data>
                        <properties>
                          <property name="Title" type="string">Announcements</property>
                          <property name="ListUrl" type="string">{listurl:Announcements}</property>
                          <property name="ChromeType" type="chrometype">Default</property>
                        </properties>
                      </data>
                    </webPart>
                  </webParts>
              </pnp:Contents>
            </pnp:WebPart>
          </pnp:WebParts>
        </pnp:Page>
      </pnp:Pages>
    </pnp:ProvisioningTemplate>
  </pnp:Templates>
</pnp:Provisioning>

**Comment by JCol79** [comment]
Template pasted weird: the important bit is...(assuming that the list exists)

        <pnp:Page Layout="TwoColumns" Overwrite="true" Url="{site}/SitePages/testIssue.aspx">
          <pnp:WebParts>
            <pnp:WebPart Title="Announcements" Row="1" Column="1">
              <pnp:Contents>
                <webParts>
                    <webPart xmlns="http://schemas.microsoft.com/WebPart/v3">
                      <metaData>
                        <type name="Microsoft.SharePoint.WebPartPages.XsltListViewWebPart,Microsoft.SharePoint, Version=15.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c" />
                        <importErrorMessage>Cannot import this Web Part.</importErrorMessage>
                      </metaData>
                      <data>
                        <properties>
                          <property name="Title" type="string">Announcements</property>
                          <property name="ListUrl" type="string">{listurl:Announcements}</property>
                          <property name="ChromeType" type="chrometype">Default</property>
                        </properties>
                      </data>
                    </webPart>
                  </webParts>
              </pnp:Contents>
            </pnp:WebPart>
          </pnp:WebParts>
        </pnp:Page>

**Comment by Didichoune** [comment]
Yes , Apply-PnPProvisioningTemplate with webparts page doesn't work.
